<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientApplication</class>
 <widget class="QMainWindow" name="clientApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>clientApplication</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="clientTabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1161</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="accountTab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QGroupBox" name="logInBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>411</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QLabel" name="emailLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>150</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLabel" name="passwordLabel">
       <property name="geometry">
        <rect>
         <x>58</x>
         <y>190</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="singInButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>250</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Sign In</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="emailField">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>150</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordField">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="loginTitle">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Existing User</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>100</y>
        <width>401</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QLabel" name="signUpTitle">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>90</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>New User</string>
       </property>
      </widget>
      <widget class="QLabel" name="fNameLabel">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>140</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>First Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="lNameLabel">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>200</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Last Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="emailLabel_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>140</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLabel" name="passLabel">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>200</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="fNameField">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>140</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lNameField">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>200</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="emailField_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>140</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="passField">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>200</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="singUpButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>260</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>50</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Welcome</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RepositoryTab">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="repoTitleLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select a Repository</string>
      </property>
     </widget>
     <widget class="QTreeView" name="fileTree">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>921</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="createFolderBox">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>220</y>
        <width>281</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="folderNameField">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string>Folder Name</string>
       </property>
      </widget>
      <widget class="QDialogButtonBox" name="createFolderButtons">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>166</width>
         <height>24</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string>Create New Folder</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="deleteFileButton">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>10</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete File</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="createFileBox">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>220</y>
        <width>281</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="createNewFileLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="fileNameField">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QDialogButtonBox" name="createFileButtons">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>166</width>
         <height>24</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </widget>
     <widget class="QListWidget" name="repoList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>191</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="createRootFolderBox">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>390</y>
        <width>281</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="rootFolderNameField">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string>Folder Name</string>
       </property>
      </widget>
      <widget class="QDialogButtonBox" name="createRootFolderButtons">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>166</width>
         <height>24</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string>Create New Folder</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1162</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewFolder">
   <property name="text">
    <string>New Folder</string>
   </property>
  </action>
  <action name="actionEditFile">
   <property name="text">
    <string>Edit File</string>
   </property>
  </action>
  <action name="actionNewTextFile">
   <property name="text">
    <string>New Text File</string>
   </property>
  </action>
  <action name="actionNewWordDocument">
   <property name="text">
    <string>New Word Document</string>
   </property>
  </action>
  <action name="actionNewExcelSheet">
   <property name="text">
    <string>New Excel Sheet</string>
   </property>
  </action>
  <action name="actionNewPowerPointPresentation">
   <property name="text">
    <string>New PowerPoint Presentation</string>
   </property>
  </action>
  <action name="actionNewRootFolder">
   <property name="text">
    <string>New Folder</string>
   </property>
  </action>
  <action name="actionViewFile">
   <property name="text">
    <string>View File</string>
   </property>
  </action>
  <action name="actionRunExecutable">
   <property name="text">
    <string>Run Executable</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
